src/encoded/types/experiment.py

```diff
@@ -156,8 +156,10 @@ class Experiment(Item):
         'biosample.biosource.individual.protected_data',
 
         # Organism linkTo
-        'biosample.biosource.individual.organism.name',
+        'biosample.biosource.organism.name',
+        'biosample.biosource.organism.scientific_name',  # not sure we need this one
         'biosample.biosource.individual.organism.scientific_name',
+        'biosample.biosource.individual.organism.name',
```

Comment:
this embed will also be redundant to `biosample.biosource.organism.name` so I think we can remove?